compute/admin_guide/runtime_defense/runtime_defense_containers.adoc

```diff
@@ -240,7 +241,7 @@ Also, consider using Prisma Cloud’s alert profile feature to alert specific te
 
 ==== Anti-malware
 
-Anti-malware provides high level control for anti-malware capabilities for containers. More granular configuration for each runtime capability is available through each the other tabs on the rule.
+Anti-malware provides high level control for anti-malware capabilities for containers. More granular configuration for each runtime capability is available through each of the other tabs on the rule.
 
 - *Prisma Cloud advanced threat protection* -- Use Prisma Cloud advanced threat protection intelligence feed, to apply malware prevention techniques across processes, networking and filesystem.
 
```

Comment:
@divyabhushan we need to add a note for k8s attacks, that Prevent is not supported for Kubernetes attacks that arise from the Network flow (because Prevent is not supported for Network at all)